FWCore/Framework/src/PathsAndConsumesOfModules.cc

```diff
@@ -141,9 +141,13 @@ namespace edm {
     using namespace edm::graph;
     //Need to lookup ids to names quickly
     std::unordered_map<unsigned int, std::string> moduleIndexToNames;
-    
+
+    std::map<std::string, unsigned int> pathStatusInserterModuleLabelToModuleID;
```

Comment:
How about switching to a `std::unordered_map`? They tend to be more memory efficient.